src/components/assets/icons/SSIEntityIcon/index.tsx

```diff
@@ -1,29 +1,41 @@
 import React, { FC } from 'react'
-import { ColorValue, View } from 'react-native'
-import Svg, { Circle, Path } from 'react-native-svg'
+import { ColorValue } from 'react-native'
+import { connect } from 'react-redux'
 
 import { entities, fonts } from '../../../../styles/colors'
+import {
+  SSIEntityIconStyled as Avatar,
+  SSITextH2LightStyled as AvatarText
+} from '../../../../styles/components'
+import { RootState } from '../../../../types'
+import { getDefaultUserNameInitials } from '../../../../utils/UserUtils'
 
 export interface IProps {
   size?: number
-  color?: ColorValue
-  backgroundColor?: ColorValue
+  fontColor?: ColorValue
+  bgColor?: ColorValue
```

Comment:
I think we should leave it as prop and not move it to styles as in future it is possible to have multiple avatars displayed in same view/list etc.